From d97a4feef81ba5bbef65ae401f8d674bc6aba58f Mon Sep 17 00:00:00 2001
From: David Milburn <dmilburn@redhat.com>
Date: Mon, 18 Aug 2014 19:58:53 -0400
Subject: [block] nvme: Define Log Page constants

Message-id: <1408391935-24886-21-git-send-email-dmilburn@redhat.com>
Patchwork-id: 87896
O-Subject: [RHEL7.1 PATCH BZ 1111259 20/22] NVMe: Define Log Page constants
Bugzilla: 1111259
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Matthew Wilcox <matthew.r.wilcox@intel.com>

Taken from the 1.1a version of the spec

Signed-off-by: Matthew Wilcox <matthew.r.wilcox@intel.com>
(cherry picked from commit 3d69bb6e4699251102dc145b7800dd012ddec375)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/nvme.h b/include/uapi/linux/nvme.h
index c611548..59e3a88 100644
--- a/include/uapi/linux/nvme.h
+++ b/include/uapi/linux/nvme.h
@@ -306,6 +306,10 @@ enum {
  NVME_FEAT_WRITE_ATOMIC = 0x0a,
  NVME_FEAT_ASYNC_EVENT = 0x0b,
  NVME_FEAT_SW_PROGRESS = 0x0c,
+ NVME_LOG_ERROR  = 0x01,
+ NVME_LOG_SMART  = 0x02,
+ NVME_LOG_FW_SLOT = 0x03,
+ NVME_LOG_RESERVATION = 0x80,
  NVME_FWACT_REPL  = (0 << 3),
  NVME_FWACT_REPL_ACTV = (1 << 3),
  NVME_FWACT_ACTV  = (2 << 3),
-- 
1.7.1